huaweicloud/services/acceptance/dcs/data_source_huaweicloud_dcs_backups_test.go

```diff
@@ -0,0 +1,155 @@
+package dcs
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDatasourceDcsBackups_basic(t *testing.T) {
+	rName := "data.huaweicloud_dcs_backups.test"
+	dc := acceptance.InitDataSourceCheck(rName)
+	name := acceptance.RandomAccResourceName()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatasourceDcsBackups_basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "backups.0.error_code", ""),
+					resource.TestCheckResourceAttrSet(rName, "backups.#"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.id"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.name"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.size"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.type"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.begin_time"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.end_time"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.status"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.description"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.is_support_restore"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.backup_format"),
+					resource.TestCheckResourceAttrSet(rName, "backups.0.progress"),
+					resource.TestCheckOutput("backup_id_filter_is_useful", "true"),
+					resource.TestCheckOutput("name_filter_is_useful", "true"),
+					resource.TestCheckOutput("status_filter_is_useful", "true"),
+					resource.TestCheckOutput("type_filter_is_useful", "true"),
+					resource.TestCheckOutput("is_support_restore_filter_is_useful", "true"),
+					resource.TestCheckOutput("backup_format_filter_is_useful", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDatasourceDcsBackups_basic(name string) string {
+	return fmt.Sprintf(`
+%s
+
+data "huaweicloud_dcs_backups" "test" {
+  depends_on  = [huaweicloud_dcs_backup.test]
+  instance_id = huaweicloud_dcs_instance.instance_1.id
+}
+
+data "huaweicloud_dcs_backups" "backup_id_filter" {
+  depends_on  = [huaweicloud_dcs_backup.test]
+  instance_id = huaweicloud_dcs_instance.instance_1.id
+  backup_id   = split("/",huaweicloud_dcs_backup.test.id).1
```

Comment:
can the following usage work? If yes, I also suggest to define the local variable as **backup_id**
```suggestion
  backup_id   = local.id
```